packages/runtime/src/widgets/Form/TextInput.tsx

```diff
@@ -81,6 +91,15 @@ export const TextInput: React.FC<TextInputProps> = (props) => {
     }
   }, [value, formInstance, values?.id, values?.label]);
 
+  useEffect(() => {
+    if (values?.mask && maskRef.current) {
+      IMask(maskRef.current.input, {
+        mask: values.mask.replace(/#/g, "0").replace(/A/g, "a"),
+        lazy: true,
+      });
+    }
+  }, [values, value, handleChange]);
```

Comment:
Fix this dependency array - remove unused dependencies and be more specific, e.g. values.mask instead of values.